client/src/App.jsx

```diff
@@ -1,53 +1,43 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-// import Counter from './Counter';
+import axios from 'axios';
 
-class App extends React.Component {
+// import TopNavBar from './components/top_bar_components/TopNavBar/TopNavBar';
+import ProductDetails from './components/product_details_components/productDetails/ProductDetails';
```

Comment:
folder naming conventions are totally individual preference, but i've personally been following Airbnb's naming conventions for folders as well as components. https://github.com/airbnb/javascript/tree/master/react#naming. TL;DR: they use camelcase for everything. Again, personal preference though!! Whatever works for you

Reply:
@karebae Thanks Karin!! I will look into airbnb style guide. I totally missed that they have naming conventions for folders. 👍